src/components/counter/Counter.tsx

```diff
@@ -2,13 +2,12 @@ import "./_counter.scss";
 
 import React, {useState} from "react";
 import classNames from "classnames";
-import {RouteComponentProps} from "@reach/router";
 
-type TCounterProps = {
+interface CounterOwnProps {
   customClassName?: string;
-} & RouteComponentProps;
+}
 
-function Counter({customClassName}: TCounterProps) {
+function Counter({customClassName}: CounterOwnProps) {
```

Comment:
I used to use `CounterOwnProps` when we were using `connect` HOC for `redux`. But as we no longer type the props coming from Redux state, we can directly use `ComponentNameProps`. For connected class components, we can still follow the "own" convention.

```suggestion
function Counter({customClassName}: CounterProps) {
```